tests/utils.py

```diff
@@ -164,3 +165,14 @@ def generate_validator_from_schema(raw_schema, **kwargs):
     schema = schema_validator(raw_schema, **kwargs)
     validator = functools.partial(validate_object, schema=schema, **kwargs)
     return validator
+
+
+@contextlib.contextmanager
+def set_env(**environ):
```

Comment:
Can you update this to use the [`monkeypatch`](https://docs.pytest.org/en/latest/monkeypatch.html) fixture provided by pytest.  It automatically cleans up anything that you patch during test runs.

Reply:
Fixed